### ISSUE TITLE
[BUG] - The latest configuration file, "conway-genesis.json", reports an error in v8.1.1.

### ISSUE THREAD

**Issue by Albert-Youngg** [IntersectMBO/cardano-node#5569]
**Summary**
The latest configuration file, "conway-genesis.json", reports an error in v8.1.1.

I downloaded the configuration file from here.
curl -O -J https://book.world.dev.cardano.org/environments/mainnet/config.json
curl -O -J https://book.world.dev.cardano.org/environments/mainnet/db-sync-config.json
curl -O -J https://book.world.dev.cardano.org/environments/mainnet/submit-api-config.json
curl -O -J https://book.world.dev.cardano.org/environments/mainnet/topology.json
curl -O -J https://book.world.dev.cardano.org/environments/mainnet/byron-genesis.json
curl -O -J https://book.world.dev.cardano.org/environments/mainnet/shelley-genesis.json
curl -O -J https://book.world.dev.cardano.org/environments/mainnet/alonzo-genesis.json
curl -O -J https://book.world.dev.cardano.org/environments/mainnet/conway-genesis.json

However, an error occurred while using cardano-node v8.1.1, causing it to fail to start.

![image](https://github.com/input-output-hk/cardano-node/assets/30194654/87a19482-3d2b-495e-8e04-bde880418243)

Completion of the proposal:
Conway related : There was an error parsing the genesis file: /ada/config/conway-genesis.json Error: "Error in $: key \"genDelegs\" not found"

**Comment by gituser** [comment]
I have the same issue with `cardano-node v8.1.2` and latest cardano-wallet.

Is there anyone who could look into this?

**Comment by zhy827827** [comment]
I have the same problem. I modified the configuration file according to the above method, but the error reported
```
Conway related : Wrong genesis file: the actual hash is "f7d46bdd3b3c8caf38351c4eef3346a89241707270be0d6106e8a407db294cc6", but the expected genesis hash given in the node configuration file is "f28f1c1280ea0d32f8cd3143e268650d6c1a8e221522ce4a7d20d62fc09783e1"
```

**Comment by smelc** [comment]
@zhy827827> in your `config.json` file, you currently have:

```json
  "ConwayGenesisHash": "f28f1c1280ea0d32f8cd3143e268650d6c1a8e221522ce4a7d20d62fc09783e1",
```

but it needs to be:

```json
  "ConwayGenesisHash": "f7d46bdd3b3c8caf38351c4eef3346a89241707270be0d6106e8a407db294cc6",
```

That's because you modified the Conway genesis file, and so its hash changed (the hash is kind of a zipped summary of the file). You must reflect this change in the `config.json` file, to acknowledge the change.

**Comment by AdmiralOOM** [comment]
After fixing the GenDelegs, it still throws error.
`Conway related : There was an error parsing the genesis file: /cardano/preview/config/conway-genesis.json Error: "Error in $: endOfInput"`

**Comment by Thunfeld** [comment]
> After fixing the GenDelegs, it still throws error. `Conway related : There was an error parsing the genesis file: /cardano/preview/config/conway-genesis.json Error: "Error in $: endOfInput"`

Yes i have the same problem, tried some stuff to fix it but nothing worked.

**Comment by smelc** [comment]
@AdmiralOOM, @Thunfeld> can you post your set of `config` files that trigger this error? So that we can try reproducing it.

**Comment by RdeWilde** [comment]
This also happens in 8.1.2.
Any update as this is a blocking production issue?

**Comment by Thunfeld** [comment]
@smelc I think that it has nothing to with te _config.json_ file.  
because everthing is right in this folder but there is somthing wrong in the cardano-node dir. 
I installed cardano-node --version 8.7.3 but for some reason it installed 8.7.1 this coukd be the issue.
also i was not able to upgrade from this version so i had to remove the hole dir.
What i dit to fix it: 
```
rm -r cardano-node
```
and
```
rm -r mainnet
```
Maybe you can get away without removing the db dir but im not sure about this.

```
git fetch --all --recurse-submodules --tags
git tag | sort -V
git checkout tags/8.7.3
```

so after a clean install everything worked again. 
make sure to check the installed version afterwards.
```
cardano-node --version
```

**Comment by aleduni** [comment]
Hello I have similar problem.

Conway related : There was an error parsing the genesis file: "/opt/cardano/cnode/files/conway-genesis.json" Error: "Error in $: key \"poolVotingThresholds\" not found"

on the config file I have 

 "ConwayGenesisFile": "conway-genesis.json", 
  "ConwayGenesisHash": "f28f1c1280ea0d32f8cd3143e268650d6c1a8e221522ce4a7d20d62fc09783e1",
and on the conway-genesis.json 
 I have 
 {
  "genDelegs": {}
}

any suggestion?
thanks

**Comment by albert-kevin** [comment]
the config files are flawed, you can fix this by using the correct config files downloaded from here
https://book.world.dev.cardano.org/environments
instead of github

for example:
base_url="https://book.world.dev.cardano.org/environments"
network="preview"
curl -O -J $base_url/$network/config.json
curl -O -J $base_url/$network/db-sync-config.json
curl -O -J $base_url/$network/submit-api-config.json
curl -O -J $base_url/$network/topology.json
curl -O -J $base_url/$network/byron-genesis.json
curl -O -J $base_url/$network/shelley-genesis.json
curl -O -J $base_url/$network/alonzo-genesis.json
curl -O -J $base_url/$network/conway-genesis.json

**Comment by gulla0** [comment]
> the config files are flawed, you can fix this by using the correct config files downloaded from here https://book.world.dev.cardano.org/environments instead of github
> 
> for example: base_url="https://book.world.dev.cardano.org/environments" network="preview" curl -O -J $base_url/$network/config.json curl -O -J $base_url/$network/db-sync-config.json curl -O -J $base_url/$network/submit-api-config.json curl -O -J $base_url/$network/topology.json curl -O -J $base_url/$network/byron-genesis.json curl -O -J $base_url/$network/shelley-genesis.json curl -O -J $base_url/$network/alonzo-genesis.json curl -O -J $base_url/$network/conway-genesis.json

Hey Kevin, that worked for me. TY!
